src/lookup_routing/readme lookup.md

```diff
@@ -0,0 +1,124 @@
+# Neural Network Substitution-based routing for the National Water Model version 3.0**
+
+## Description
+The National Water Model (NWM) is a hydrologic model that simulates observed and forecast
+streamflow of approximately 2.7 million stream segments in the continental US (CONUS). The
+channel routing computation of the current operational NWM is based on the Muskingum-Cunge
+method (MC), which solves its two parameters using a numerical trial-and-error method called
+the secant method. These iterations can be computationally expensive, requiring approximately one
+core hour to complete five minutes of simulation-time the channel routing calculations in the model..
+We propose the use of a machine learning method to solve nonlinear partial differential routing
+equations. By training a neural network to reproduce the hydraulic routing computation, we
+expect to generate ensembles of predictions that are interchangeable with the output of the
+traditional method in a fraction of the time required. For example, elements of the network that
+are currently in a receding condition are expected to quickly reach a solution due predictable
+patterns of relaxation. By accelerating the rate in which the NWM’s hydraulic routing
+computation is performed, we explore the impact to the overall computational burden of the
+model.
+We have created a recurrent neural network model that imitates the output of a formulated MC
+single segment and Manning’s equation. Additionally, we have confirmed parity of the single
+segment MC calculation with the native WRF-Hydro version. Evaluations of the impacts of each
+input parameter to the MC single segment calculation were also assessed during this process.
+We developed modifications to the model to make the prototyping of parameters more efficient.
+Finally, we explored if the application of neural networks could be applied to other
+computationally heavy processes in the NWM.
+
+#Installation
+Install Python3
+Install Tensorflow
+Install Keras
+Run the file located at  https://github.com/NOAA-OWP/t-route/blob/master/src/fortran_routing/mc_pylink_v00/MC_singleSeg_singleTS/mc_sseg_stime_NOLOOP_demo.py to generate a required file.
+
+## Usage
+Substitute for MC single segment
+
+## Components
+-=Main=- NNFullCombos.py :

```

Comment:
Not quite sure what the significance (if any) of the `-==-` is... Whatever the command should be, wrap that in `backticks`.